src/github/code-scanning-alert.ts

```diff
@@ -0,0 +1,15 @@
+import transformCodeScanningAlert from "../transforms/code-scanning-alert";
+import { Context } from "probot/lib/context";
+import logger from "../config/logger";
+
+export default async (context: Context, jiraClient): Promise<void> => {
```

Comment:
Should we put it behind a feature flag?

Reply:
I wasn't aware GitHub App supported feature flagging, we should add this yep.

We should be careful to rollout by tenant and not by request (and be careful in our email to accept new permissions)